website/frontend/templates/docs/tags.html

```diff
@@ -105,21 +105,33 @@ <h1 id="basic_tags">Basic Tags</h1>
           <td>date</td>
           <td> <a href="http://musicbrainz.org/doc/Release_Date" title="Release Date">Release Date</a> (YYYY-MM-DD)</td>
         </tr>
+        <tr>
+          <td>originalartist</td>
+          <td>Original <a href="http://musicbrainz.org/doc/Track_Artist" title="Track Artist">Track Artist</a> of the earliest release in the <a href="http://musicbrainz.org/doc/Release_Group" title="Release Group">Release Group</a> intended for the performer(s) of the original recording.</td>
```

Comment:
Links to MusicBrainz should use HTTPS. This is not done for the current links either, but would be a nice additional commit to have them all fixed.